week-3/D-array-filter/exercise.js

```diff
@@ -8,7 +8,11 @@
 
 var pairsByIndexRaw = [[0, 3], [1, 2], [2, 1], null, [1], false, "whoops"];
 
-var pairsByIndex; // Complete this statement
+function isValid(validContent){
+    return validContent;
```

Comment:
Hmm not actually doing any validation here. How can you return true only for arrays of length 2